src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountMoreThreeDays = 20;
+  const discountMoreSevenDays = 50;
+  const rentalCost = 40;
+
+  let price = days * rentalCost;
```

Comment:
There is no need in `let` variable here, use `const` instead you can just put return statements in your if blocks and use this variable right in return